docs/source/routing/request-lifecycle.mdx

```diff
@@ -0,0 +1,288 @@
+---
+title: Router Request Lifecycle
+subtitle: Understand how the router processes client requests
+description: Understand how GraphQL client requests get processed by the request lifecycle pipeline of Apollo GraphOS Router.
+---
+
+import RequestLifecycleOverviewDiagram from '../../shared/diagrams/router-request-lifecycle-overview.mdx';
+
+## The request lifecycle
+
+Every client request made to a GraphOS Router goes through its **router request lifecycle**: a multi-stage pipeline of services that processes requests and returns responses.
+
+<RequestLifecycleOverviewDiagram />
+
+A client request is processed first by passing it between services along the lifecycle's **request path** to create and send its constituent subgraph requests. Then all the responses from the subgraph requests are gathered into a client response along the lifecycle's **response path**.
+
```

Comment:
Before getting into the nitty-gritty in the following sections, is it worth calling out why users should _care_ about the lifecycle? (I.e. for instrumentation and customization)? I imagine most folks will come to this page because of these needs, but for the rare viewers who are browsing the top-level pages (About GraphOS Router, Configure Your Router, Request Lifecycle), it may be worth leading with this info.